packages/frontend/src/components/RuntimeAdapter.tsx

```diff
@@ -52,6 +59,10 @@ export default function RuntimeAdapter({
           await window.__selectAccount(notificationAccountId)
         } else if (chatId !== 0) {
           await selectChat(accountId, chatId)
+          if (isWebxdcInfo) {
+            const message = await BackendRemote.rpc.getMessage(accountId, msgId)
+            internalOpenWebxdc(accountId, message)
+          }
```

Comment:
It's unfortunate that we have to do an RPC call here, and then another one inside of `internalOpenWebxdc`.

Would be nice to optimize it at some point. Doesn't have to be done in this MR I guess.

Reply:
Decided to not open apps on notification click, so this will be resolved.

Reply:
The second call is always needed if we have a click on a WebxdcInfoMessag  to get the webxdcInfo from the parent message. No way to get it it in one call